drivers/storage/floppy_new/floppy.c

```diff
@@ -2516,10 +2516,11 @@ Return Value:
 
 #ifndef __REACTOS__
             context = ExAllocatePool(NonPagedPoolNx,
+                                     sizeof(COMPLETION_CONTEXT));
```

Comment:
Useless change, revert

Reply:
On AppVeyor, builds fail without this change(s). Does that not happen for you?

Reply:
The version that doesn't duplicate the line will fail iff ExAllocatePool is a function-style macro. That will be the case if `classpnp.h` is included and `POOL_TAGGING` is defined... which seems to be the case by default. So the change seems to make sense.